qwerty_pyrt/rust/src/wrap_ast.rs

```diff
@@ -7,7 +7,7 @@ use pyo3::{
 };
 use qwerty_ast::{ast, dbg, typecheck};
 use std::fmt;
-
+use qwerty_ast::typecheck::{TypeEnv as RustTypeEnv, typecheck_expr};
```

Comment:
Style opinion, but I think this makes it harder to grep for things in / trace through the code
```suggestion
use qwerty_ast::typecheck;
```